package.json

```diff
@@ -213,6 +214,7 @@
 		"@automattic/calypso-eslint-overrides": "workspace:^",
 		"@automattic/calypso-jest": "workspace:^",
 		"@automattic/eslint-plugin-json": "workspace:^",
+		"@automattic/generators": "workspace:^",
```

Comment:
Noting that as an external user / spectator / consumer, the name `@automattic/generators` adds a vagueness, since it's not immediately clear what the package will generate. Should it be `@automattic/package-generators`? One more thing - why the plural? I'd consider the idea being having one generator, with multiple routes for generating packages, but still, one generator. WDYT?

Reply:
Sounds good! I named the package to `@automattic/calypso-package-generator`. WDYT?

Reply:
Thanks! My opinion is that the `calypso-` part could feel redundant, but I can see why it can be useful, so leaving this to your judgment.